code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -181,7 +181,7 @@
 		if((!silence_steps || shoe_sound) && TR.use(4))
 			silence_steps = 1
 			shoe_sound = null
-			to_chat(user, "You tape the soles of [src] to silence their footsteps.")
+			to_chat(user, "You tape the soles of [src] to silence them.")

```

Comment:
I think this would be better as `to silence your footsteps`, to make it clear what this is doing.

Reply:
Funnily enough I originally had it like that, except if they're not on your feet it doesn't really apply. But that's a fringe case.

Reply:
`their` referring to the shoes, not the person.

I think the original or BT's change is better than Tiger's suggestion for the above stated reason.